CHANGELOG.md

```diff
@@ -2,17 +2,19 @@
 
 ## Unreleased
 
+## 3.13.1 (Patch release)
+
 ### Fixes
 
 We’ve made fixes to GOV.UK Frontend in the following pull requests:
 
-- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue.
-- [#2265: Don’t remove focus outline from disabled link buttons in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2265)
-- [#2273: Fix invisible footer OGL logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue.
+- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```

Comment:
I think it should either be 'forced-color mode' or 'forced colors mode' but definitely double check this :)

Reply:
Great spot! Just looked up [Chrome](https://www.chromestatus.com/feature/5757293075365888#:~:text=Adds%20the%20'forced%2Dcolors',color%20palette%20on%20the%20page.) and [Mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/forced-color-adjust), and yep, 'forced colors mode' it is..I'll make the needed changes here and then reassign you the review if that's ok. Thanks again!

Reply:
```suggestion
- [#2264: Improve focus state for radio and checkbox controls in forced colors mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```